client/my-sites/theme/main.jsx

```diff
@@ -9,6 +9,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import page from 'page';
+import { isPremium } from 'my-sites/themes/helpers';
```

Comment:
> Incidentally, the above doesn't work since getForumUrl uses this.isPremium().

Mhm, noticed last night. Going to ES6ify `helpers`, probably as part of #5284.